### ISSUE TITLE
Ospray Deployment

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#1807]
Follows #1767 and #1806

**Comment by dcwhite** [comment]
@jessdtate @xuanhuang1 I'll make this the official "clean-up" issue. Look for completion of this before trying any changes to the core Ospray code.

**Comment by dcwhite** [comment]
Post Qt5--I'll bump this up to X though

**Comment by dcwhite** [comment]
@jessdtate A good one to try to fit into a CMake/CPack training session.

**Comment by jessdtate** [comment]
Yeah, I think that's a great example to try.  I think we have time next week.

**Comment by dcwhite** [comment]
Procedure (takes CPack out of the equation):
- [ ] run make install into clean directory
- [ ] copy dir to clean environment (docker container?)
- [ ] check if everything loads
- [ ] fix rpath errors

**Comment by dcwhite** [comment]
**Analysis (2026-06-13)**

OSPRay deployment via CPack / make install. The checklist in comments (run make install into clean dir → copy to clean environment → verify loading → fix rpath errors) is the right approach.

Current status: OSPRay integration is disabled in CI and the rpath/install story for the OSPRay dylibs is not wired into the SCIRun CPack config. The main challenge is that OSPRay ships several shared libs that need either embedding in the app bundle or fixup via install_name_tool.

Blocked on the same OSPRay upgrade decision as #1703 and #1762. Docker container testing (mentioned in comments) is the cleanest way to verify portability — ties in naturally with #1720.
